### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by swapnil1993**

### CODE REVIEW

File: api-reference/beta/resources/termcolumn.md [hunk 1]
```diff
@@ -15,11 +15,16 @@ Indicates that the column's values contains taxonomy data.
...
-| parentTerm     | microsoft.graph.termStore.term | Specifies the term guid whose children can be selected as column's value.  
+| allowMultipleValues | Boolean | Specifies whether the column will allow more than one value.
 | showFullyQualifiedName | Boolean | Specifies whether to display the entire term path or only the term label.  
+
+## Relationships
+
+| Relationship   | Type                      | Description
+|:----------------|:--------------------------|:-------------------------------
+| parentTerm     | microsoft.graph.termStore.term | Specifies the term guid whose children can be selected as column's value.
 | termSet      | microsoft.graph.termStore.set | Termset whose children can be selected as column's value.
```
Comment (isvargasmsft) [inline api-reference/beta/resources/termcolumn.md:28]: These 2 relationships are not defined in the metadata. Are these about to be introduced?

File: api-reference/v1.0/resources/documentset.md [hunk 2]
```diff
@@ -15,16 +15,21 @@ Represents a document set in SharePoint.
...
-| Property name  | Type    | Description|
+| Property  | Type    | Description|
 |:---------------|:--------|:--------------------------------------------------|
 | shouldPrefixNameToFile | Boolean  | Add the name of the document set to each file name.|
 | allowedContentTypes | Collection(microsoft.graph.contentTypeInfo) | Content types allowed in document set.|
 | defaultContents     | Collection(microsoft.graph.documentSetContent) | Default contents of document set. | 
 | propagateWelcomePageChanges | Boolean | Specifies whether to push welcome page changes to inherited content types.  |
-| sharedColumns       | Collection(microsoft.graph.columnDefinition) | Columns edited on the document set that synchronize to all documents in the set. These are read-only on the documents themselves. |
-| welcomePageColumns  | Collection(microsoft.graph.columnDefinition)  | Specifies columns to show on the welcome page for the document set.  |
 | welcomePageUrl      | string | Welcome page absolute URL.  |
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/documentset.md:24]: Please order these properties alphabetically.

File: api-reference/beta/resources/termcolumn.md (hunk 1 above)
Comment (isvargasmsft) [reply api-reference/beta/resources/termcolumn.md:28]: I noticed these properties are defined in v1 but not in beta. Please review and fix accordingly.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello, 

Thank you very much for your work in this updates. 

Please take time to review 2 main issues:
- Please order the properties in alphabetically in the Properties table. 
- Please address some inconsistencies in the metadata. 

Feel free to ping me on the side if you have any questions. 

Regards, 

Isaac Vargas

File: api-reference/beta/resources/termcolumn.md (hunk 1 above)
Comment (kanchm) [reply api-reference/beta/resources/termcolumn.md:28]: api-reference/beta/resources/termcolumn.md(18,1): [Microsoft.FileServices] Remove 'parentTerm, termSet' from the documentation or add it to the schema.

Comment (swapnil1993) [reply api-reference/beta/resources/termcolumn.md:28]: I will be adding it to beta schema

Comment (swapnil1993) [reply api-reference/beta/resources/termcolumn.md:28]: I have checked in the changes to update csdl, it should be fully deployed in next few days

**Comment by swapnil1993** [comment]
#feedback-addressed

File: api-reference/v1.0/resources/documentset.md [hunk 3]
```diff
@@ -15,16 +15,21 @@ Represents a document set in SharePoint.
...
+## Relationships
+
+| Relationship   | Type                      | Description
+|:----------------|:--------------------------|:-------------------------------
+| sharedColumns       | Collection(microsoft.graph.columnDefinition) | Columns edited on the document set that synchronize to all documents in the set. These are read-only on the documents themselves. 
+| welcomePageColumns  | Collection(microsoft.graph.columnDefinition)  | Specifies columns to show on the welcome page for the document set.
+
 ## JSON representation
 
 The following is a JSON representation of a **documentSet** resource.
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/documentset.md:35]: Please update the JSON representation to reflect the actual structure of the resource.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Please update the JSON representation sections to reflect the changes and remove the nav properties.

**Comment by swapnil1993** [comment]
#feedback-addressed

**Review by isvargasmsft — APPROVED** [review]

File: api-reference/beta/resources/documentset.md [hunk 4]
```diff
@@ -16,16 +16,21 @@ Contains metadata about document set settings.
...
-| Property name  | Type    | Description
+| Property  | Type    | Description
 |:---------------|:--------|:--------------------------------------------------
-| shouldPrefixNameToFile | Boolean  | Add the name of the Document Set to each file name.
 | allowedContentTypes | Collection(microsoft.graph.contentTypeInfo) | Content types allowed in document set.
 | defaultContents     | Collection(microsoft.graph.documentSetContent) | Default contents of document set.  
 | propagateWelcomePageChanges | Boolean | Specifies whether to push welcome page changes to inherited content types.  
-| sharedColumns       | Collection(microsoft.graph.columnDefinition) | Columns edited on the document set that synchronize to all documents in the set. These are read-only on the documents themselves. 
-| welcomePageColumns  | Collection(microsoft.graph.columnDefinition)  | Specifies columns to show on the welcome page for the document set.  
+| shouldPrefixNameToFile | Boolean  | Add the name of the Document Set to each file name.
```
Comment (Lauragra) [inline api-reference/beta/resources/documentset.md:24]: This description is unclear. It's a Boolean property, so I would expect a description such as: "Indicates whether to add the name of the document set to each file name." You might also want to indicate whether the default value is `true` or `false`.

File: api-reference/beta/resources/termcolumn.md [hunk 5]
```diff
@@ -15,11 +15,16 @@ Indicates that the column's values contains taxonomy data.
...
-| allowMultipleValues | Boolean | Specifies whether the column will allow more than one value   
-| parentTerm     | microsoft.graph.termStore.term | Specifies the term guid whose children can be selected as column's value.  
+| allowMultipleValues | Boolean | Specifies whether the column will allow more than one value.
 | showFullyQualifiedName | Boolean | Specifies whether to display the entire term path or only the term label.  
+
+## Relationships
+
+| Relationship   | Type                      | Description
+|:----------------|:--------------------------|:-------------------------------
+| parentTerm     | microsoft.graph.termStore.term | Specifies the term guid whose children can be selected as column's value.
```
Comment (Lauragra) [inline api-reference/beta/resources/termcolumn.md:27]: GUID is an acronym; please capitalize. Suggested revision: "Specifies the GUID of the parent term for which the child terms can be selected as the column value."

File: api-reference/v1.0/resources/documentset.md [hunk 6]
```diff
@@ -15,16 +15,21 @@ Represents a document set in SharePoint.
...
-| Property name  | Type    | Description|
+| Property  | Type    | Description|
 |:---------------|:--------|:--------------------------------------------------|
-| shouldPrefixNameToFile | Boolean  | Add the name of the document set to each file name.|
 | allowedContentTypes | Collection(microsoft.graph.contentTypeInfo) | Content types allowed in document set.|
 | defaultContents     | Collection(microsoft.graph.documentSetContent) | Default contents of document set. | 
 | propagateWelcomePageChanges | Boolean | Specifies whether to push welcome page changes to inherited content types.  |
-| sharedColumns       | Collection(microsoft.graph.columnDefinition) | Columns edited on the document set that synchronize to all documents in the set. These are read-only on the documents themselves. |
-| welcomePageColumns  | Collection(microsoft.graph.columnDefinition)  | Specifies columns to show on the welcome page for the document set.  |
+| shouldPrefixNameToFile | Boolean  | Add the name of the document set to each file name.|
```
Comment (Lauragra) [inline api-reference/v1.0/resources/documentset.md:23]: Please fix the description here too.

**Comment by swapnil1993** [comment]
#feedback-addressed

File: api-reference/beta/resources/documentset.md (hunk 4 above)
Comment (swapnil1993) [reply api-reference/beta/resources/documentset.md]: Made the description change. Booleans are false by default therefore skipping specifying the default value

**Pull request merged**